src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -453,7 +429,10 @@ static inline int MPIDI_OFI_win_init(MPI_Aint length,
     window_instance =
         MPIDI_OFI_index_allocator_alloc(MPIDI_OFI_COMM(win->comm_ptr).win_id_allocator,
                                         MPL_MEM_RMA);
-    MPIDI_OFI_WIN(win).win_id = ((uint64_t) comm_ptr->context_id) | (window_instance << 32);
+
+    MPIDI_OFI_RMA_KEY_PACK(MPIDI_OFI_WIN(win).win_id, (win->comm_ptr->context_id),
```

Comment:
In the original code, `context_id` was used like this:
```C
MPIR_CONTEXT_READ_FIELD(PREFIX, comm_ptr->context_id);
```
Should we follow a similar way?

Reply:
That makes sence for 16-bit RMA-keys only, Anyway it is hidden in MPIDI_OFI_rma_key_pack now.